packages/configure/index.js

```diff
@@ -0,0 +1,22 @@
+var compose = require('@stamp/compose');
+var Privatize = require('@stamp/privatize');
+
+function configure(_, options) {
+  var configuration = options.stamp.compose.configuration;
+  var deepConfiguration = options.stamp.compose.deepConfiguration;
+  this.config = Object.freeze(
+    Object.assign({}, configuration, deepConfiguration)
```

Comment:
Not sure really, it only seemed logical as deep is usually more complex and worst to lose. I can change it, but I don't think it matters that much considering that you usually want to avoid conflicts.